app/models/spree/adyen/notification_processor.rb

```diff
@@ -143,6 +143,12 @@ def create_missing_payment
         payment
       end
 
+      def payment_method_for(order)
+        order.store.payment_methods.where(type: 'Spree::Gateway::AdyenHPP', active: true).last!
+      rescue
```

Comment:
I'm not a big fan of rescuing everything, I would prefer something like:
`order.store.payment_methods.active.find_by(type: "Spree::Gateway::AdyenHPP") || Spree::Gateway::AdyenHPP.last`